### ISSUE TITLE
deployment to github releases from appveyor fails

### ISSUE THREAD

**Issue by anthrotype** [greginvm/pyclipper#18]
The upload of the generated wheels to Github Releases requires that the admin of this repo, @greginvm, updates the `auth_token` field in the .appveyor.yml file with his Github API key, as explained in the inline comments in that file.

I am only a collaborator so I can push tags, but can't generate a new API token, nor I can encrypt it using the Appveyor account that is linked to this repo (that also needs @greginvm privileges).

I am thinking that maybe it's not worth keeping this "upload to github" feature. We simply need the wheels to be uploaded to PyPI, where pip can download them from. Having them on Github is a bonus but not really required.

So if we agree, I can set up Appveyor and Travis to upload the wheels directly to PyPI using my own PyPI credentials (i have access to the pyclipper PyPI account so it should work).

**Comment by anthrotype** [comment]
for now I manually downloaded the wheels from appveyor, and uploaded them manually to both Github and PyPI

**Comment by greginvm** [comment]
Hey, @anthrotype we talked once that I'll transfer the repo to you. I just tried but I get an error because you have a fork of it. Still want to accept?

<img width="348" alt="image" src="https://user-images.githubusercontent.com/10724016/43217827-f6efa17e-9042-11e8-9cef-b62c05efea7b.png">

**Comment by anthrotype** [comment]
@greginvm yes, that's my fork.. Actually, I'd rather you transfer it to the fonttools organization:
https://github.com/fonttools

**Comment by anthrotype** [comment]
and thank you! :heart:

**Comment by greginvm** [comment]
Hi, I can't transfer to fonttools because I don't have the "create repository" permission. We have 2 options, either I transfer it to you and then you transfer it to fonttools, or you give me create repo permissions in fonttools. Whichever options suits you better :)

https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/#transferring-to-an-organization

<img width="490" alt="image" src="https://user-images.githubusercontent.com/10724016/43218089-b735f802-9043-11e8-8faa-80e4abb2015b.png">

**Comment by anthrotype** [comment]
you are now a member ;)

**Comment by greginvm** [comment]
Nice! :) Repo transfered, tell me if you need anything

**Comment by anthrotype** [comment]
thanks! :1st_place_medal:

**Comment by greginvm** [comment]
Can you do the auto upload to pypi now?

**Comment by anthrotype** [comment]
yes, I'm admin now. I'll do when the time comes to make a new release. The latest is already on PyPI.

**Comment by greginvm** [comment]
OK, great, i hope it will serve you well 😸 ! If you want you can now remove me from the fonttools org.

**Comment by anthrotype** [comment]
you can stay if you wish :)

**Comment by greginvm** [comment]
Well, I don't work on pyclipper anymore, so I don't need the membership. Thanks anyway!
